wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -142,7 +142,8 @@ def _tokenize(self, X):
 
     def _prep_data_for_prediction(self, X):
         X_tokenized = self._tokenize(X)
-        return tf.convert_to_tensor(self.model.predict(X_tokenized))
+        predictions = self.model.predict(X_tokenized)[0]  # Issue
```

Comment:
fyi what I did to line 238 in predict `self.score(X).argmax(axis=2).flatten()` means that taking the 0th element will be just one prediction now, not the array on predictions for all of X. The flatten() should have sorted the output of predict out so it is indeed just "An array of 0s and 1s" like the prediction function docstring describes.